app/components/Settings/Settings.react.js

```diff
@@ -134,6 +138,9 @@ class Settings extends Component {
             if (this.state.editMode) this.setState({editMode: false});
             if (this.props.connectionNeedToBeSaved) this.props.setConnectionNeedToBeSaved(false);
         }
+        if (nextProps.connectorSettingsRequest.status === 200 && this.props.connectorSettingsRequest.status !== 200) {
+            this.setState({urls: nextProps.connectorSettingsRequest.content});
```

Comment:
does a request to settings return all of the settings or just the `urls`? the name of the endpoint and the actions indicate that it returns all of the settings but the assignment here indicates that it just returns the `urls`